### PR TITLE
Expand the size analysis cli to support bundle size analysis

### PR DESCRIPTION
**Pull request by Feiyang1**
Added a `bundle` command to the size analysis tool. It takes an array of bundle definitions (basically imports from packages) and analyze their size using either rollup, webpack or both.

As opposed to analyzing the size of each individual exports from a module, this command is useful for analyzing the size of a combination of exports from different modules.

Sample bundle definition file: 
```
[
    {
        "name": "test",
        "dependencies": [
            {
                "packageName": "@firebase/app",
                "versionOrTag": "exp",
                "imports": [
                    "initializeApp"
                ]
            },
            {
                "packageName": "@firebase/firestore",
                "versionOrTag": "exp",
                "imports": [
                    "getFirestore",
                    "doc",
                    "getDoc"
                ]
            }
        ]
    }
] 
```

Run command on this file: 
`yarn sa bundle -i repo-scripts/size-analysis/bundle-definition-examples/bundle-definition-1.json -o res -b both`

output: 
```
[
  {
    "name": "test",
    "results": [
      {
        "bundler": "rollup",
        "size": 148098,
        "gzipSize": 48871
      },
      {
        "bundler": "webpack",
        "size": 192989,
        "gzipSize": 62322
      }
    ]
  }
]
```

### CODE REVIEW

File: packages/util/src/util.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/**
```
Comment (hsubox76) [inline packages/util/src/util.ts:1]: A file named util in util is kind of weird, maybe "formatters.ts"?

File: repo-scripts/size-analysis/cli.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      inputModule: {
+        type: 'array',
+        alias: 'im',
+        desc:
+          'The name of the module(s) to be analyzed. example: --inputModule "@firebase/functions-exp" "firebase/auth-exp"'
+      },
+      inputDtsFile: {
+        type: 'string',
+        alias: 'if',
+        desc: 'support for adhoc analysis. requires a path to dts file'
```
Comment (hsubox76) [inline repo-scripts/size-analysis/cli.ts:37]: Is this a `d.ts` file or is `dts` something else?

**Review by hsubox76 — APPROVED** [review]

File: repo-scripts/size-analysis/cli.ts (hunk 2 above)
Comment (Feiyang1) [reply repo-scripts/size-analysis/cli.ts:37]: `d.ts` file. Updated to `d.ts` to make it clear.

File: packages/util/src/util.ts (hunk 1 above)
Comment (Feiyang1) [reply packages/util/src/util.ts:1]: Nice. renamed to `formatters.ts`

**Pull request merged**